saved-to-the-uttermost/updated/02-chapter-1.adoc

```diff
@@ -267,11 +273,13 @@ But that infants cannot be counted among these guilty
 ones is plainly shown in the following verse:
 "`But sin is not imputed where there is no law.`"
 
-Objection: Their second objection is from Ps. 51:5, "`Behold,
+[.discourse-part]
+__Objection:__ Their second objection is from Ps. 51:5, "`Behold,
```

Comment:
You don't need to change this, but for future reference, using the `[.discourse-part]` the words "Question:", "Answer:", "Objection:", and "Inquiry:" are all _automatically italicized_. You only need to manually italicize the beginning word if it's something non-standard (like the Tenant/Landlord example on the styleguide, linked below:)

See https://flp-styleguide.netlify.com/#discourse

Reply:
yeah... that must have just been an error. I don't think I did the rest with manual italics.